### PR TITLE
minor change to workaround globalize bug, fixes 

### PR DESCRIPTION
**Pull request by alepore**
this has no impact on spree but it's the only way to workaround a pretty bad bug of spree_i18n (caused by globalize): https://github.com/spree-contrib/spree_i18n/issues/597

should also go on 3-0-stable, at least

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://alepore-globalize-workaround.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://alepore-globalize-workaround.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: core/app/models/spree/product_property.rb [hunk 1]
```diff
@@ -18,7 +18,8 @@ def property_name
 
     def property_name=(name)
       unless name.blank?
-        unless property = Property.find_by(name: name)
+        # don't use `find_by :name` to workaround globalize/globalize#423 bug
+        unless Property.where(name: name).first
```
Comment (JDutil) [inline core/app/models/spree/product_property.rb]: Shouldn't this remain `property = Property.where`?

Comment (alepore) [inline core/app/models/spree/product_property.rb]: oh sure, bad pasting.
i've made some refactor for simpler reading now

**Comment by JDutil** [comment]
Thanks

**Pull request merged**